e2store/src/bin/era2-stats.rs

```diff
@@ -61,6 +78,24 @@ fn main() -> anyhow::Result<()> {
             };
             stats.storage_entry_count += 1;
             stats.storage_item_count += storage.len();
+
+            if index + 1 < account.storage_count {
+                assert_eq!(
+                    storage.len(),
+                    10_000_000,
+                    "Non-final storage entry doesn't have 10M items",
+                );
+            }
```

Comment:
It took me a minute to grok this (granted, I was unfamiliar with the era format). Maybe it's the double-negative in the error message? A quick comment might help speed up comprehension for era noobs. Something like:
```suggestion
            // Validate that every chunk of storage is full, with 10M items (skipping the final chunk)
            if index + 1 < account.storage_count {
                assert_eq!(
                    storage.len(),
                    10_000_000,
                    "Non-final storage entry doesn't have 10M items",
                );
            }
```

Reply:
Added comment, removed double-negative and added more details to the message.